stylus/include/WhereToFindUs.styl

```diff
@@ -0,0 +1,51 @@
+.WhereToFindUs
+	margin-bottom: 3rem
+	margin-top: 5rem
+	text-align: center
+	>h1
+		font-family: Arvo
+		margin-bottom: 3rem
+	>p
+		margin-right: 33%
+		margin-left: 33%
+	>div:nth-child(3)
+		display: flex
+		flex-wrap: wrap
+		margin-left: 8rem
```

Comment:
Avoid setting asymmetrical margins like this. It might make things look right in the moment, but it tends to cause problems. Setting margins on both sides and using `justify-content: center` should work fine.